networkit/_NetworKit.pyx

```diff
@@ -706,6 +707,18 @@ cdef class Graph:
 	 	"""
 		return self._this.addNode()
 
+	def addNodes(self, numberOfNewNodes):
```

Comment:
In C++ `numberOfNewNodes` is 1 by default, in Python we should have the same behavior.

Reply:
Actually I did the opposite: removed the default in C++'s implementation. If you only want to add a single node every time, you should directly use `addNode`